tests/TestCase.php

```diff
@@ -85,6 +85,20 @@ protected function getEnvironmentSetUp($app)
                     PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
                 ]) : [],
             ],
+            'database.connections.pgsql' => [
```

Comment:
Did we not have a pgsql connection before? Or are we setting something differently here

Reply:
We had a pgsql connection before, and now, I see we don't really need to set this here – the config is getting set in the PostgresTenancyBootstrapper, so I don't think there's a reason to keep this.